editor-components/content/content10/content10.module.scss

```diff
@@ -1,173 +1,202 @@
+                }
+            }
+        }
+    }
 
     .max-content {
-        max-width: var(--composer-content-width);
-        margin: auto;
         display: flex;
-        gap: 30px;
-        padding: 50px 0;
-        @media screen and (max-width: 992px) {
+        @media screen and (max-width: $composer-tablet-width-lg) {
             flex-direction: column;
             gap: 0;
             align-items: center;
+            margin-bottom: var(--composer-gap-md);
+            
         }
 
         .cards {
-            display: flex;
-            flex-wrap: wrap;
+            &:before{
+                content: "";
+                position: absolute;
+                right: 0px;
+                top: 0px;
+                width: 2000px;
+                background: color-mix(in srgb, var(--composer-html-background), var(--composer-font-color-primary) 5%);;
+                height: 100%;
+                z-index: 0;
+            }
+            position: relative;
+            border-radius: var(--composer-border-radius-sm);
             width: 100%;
-            background: var(--composer-primary-color);
-            padding: 70px 60px;
+            background: color-mix(in srgb, var(--composer-html-background), var(--composer-font-color-primary) 5%);
             height: fit-content;
+            padding: var(--composer-gap-xl) calc(var(--composer-gap-xl) + 20px) var(--composer-gap-xl) 0px;
+            margin-bottom: var(--composer-gap-lg);
 
-            @media screen and (max-width: 1200px) {
-                padding: 70px 120px;
+            @media screen and (max-width: $composer-tablet-width-lg) {
+                padding: var(--composer-gap-md);
+                margin-bottom: 0px;
             }
 
-            @media screen and (max-width: 992px) {
-                padding-bottom: 330px;
-            }
-
-            @media screen and (max-width: 768px) {
-                flex-direction: column;
-                .card {
-                    width: 100% !important;
-                }
-            }
-
-            @media screen and (max-width: 600px) {
-                padding-left: 15px;
-                padding-right: 15px;
+            @media screen and (max-width: $composer-phone-width) {
+                padding: 0px;
             }
 
             .card {
-                width: 50%;
+                z-index: 1;
                 display: flex;
-                padding: 24px;
-                gap: 1rem;
                 transition: 0.4s;
+                color: var(--composer-font-color-primary);
+                padding: var(--composer-gap-md);
+                gap: var(--composer-gap-md);
+                border-radius: var(--composer-border-radius-sm);
 
                 .icon-box {
-                    min-width: 45px;
-                    height: 45px;
+                    min-width: calc(2.67rem + var(--composer-gap-xs));
+                    height: calc(2.67rem + var(--composer-gap-xs));
+
```

Comment:
when the font size changed the size of the icon-box also will change. is that what you want?